packages/tasks/src/local-apps.ts

```diff
@@ -302,6 +339,13 @@ export const LOCAL_APPS = {
 			(model.pipeline_tag === "text-generation" || model.pipeline_tag === "image-text-to-text"),
 		snippet: snippetVllm,
 	},
+	"mlx-lm": {
+		prettyLabel: "MLX LM",
+		docsUrl: "https://github.com/ml-explore/mlx-lm",
+		mainTask: "text-generation",
+		displayOnModelPage: isMlxModel,
```

Comment:
i would only display it on actually supported models (so check the pipeline_tag)

Reply:
(note, mainTask does not check the model's task on a model page, it's only used for the settings page)